File: eRecipes/UI/erecipes_mobile/lib/models/transakcija_log25062025.dart
import 'package:erecipes_mobile/models/korisnik.dart';
import 'package:json_annotation/json_annotation.dart';

part 'transakcija_log25062025.g.dart';

@JsonSerializable()
class TransakcijaLog25062025 {
  int? transakcijaLog25062025Id;
  String? staraVrijednost;
  String? novaVrijednost;
  DateTime? vrijemePromjene;
  Korisnik? korisnik;
  int? korisnikId;

  TransakcijaLog25062025();

  //factory TransakcijaLog25062025.FromJson(Map<String, dynamic> json) =>
   //   _$TransakcijaLog25062025FromJson(json);

 // Map<String, dynamic> toJson() => _$TransakcijaLog25062025ToJson(this);
}

File: eRecipes/eRecipes.Service/Database/TransakcijaLog25062025.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eRecipes.Service.Database
{
    public class TransakcijaLog25062025
    {
        public int TransakcijaLog25062025Id { get; set; }
        public string StaraVrijednost {  get; set; }
        public string NovaVrijednost { get; set; }
        public DateTime VrijemePromjene { get; set; }
        public Korisnik Korisnik { get; set; }
        public int KorisnikId { get; set; }

    }
}
